views/includes/global-header.njk

```diff
@@ -24,7 +24,32 @@
     <div class="global-header__inner">
         <div class="global-header__extra">
             <ul class="global-header__navigation-secondary">
-                <li><a href="{{ globalNavigation.home.url }}">{{ globalNavigation.home.label }}</a></li>
+                <noscript>
```

Comment:
This is how this renders with JS off:

![image](https://user-images.githubusercontent.com/394376/66655863-c5a6ac80-ec34-11e9-8291-6991d9d2a3cd.png)

I should have been clearer, sorry! My solution posted here didn't require the use of `<noscript>` anymore – the `.js-hidden` classes will set `display: none` **for users with JavaScript enabled** so users without JavaScript will still see them, which is why the Log Out link has an extra class of `u-hidden` which hides it for all users (until the JS makes it visible again). 

You can delete the `<noscript>` block here :)

Reply:
Ahh, you're right!

It was my bad tbh. I understood you earlier and knew the intention as well, just forgot to remove that bit :) Updated now!